consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/network/IbftMessageTransmitter.java

```diff
@@ -47,42 +53,57 @@ public void multicastProposal(
       final ConsensusRoundIdentifier roundIdentifier,
       final Block block,
       final Optional<RoundChangeCertificate> roundChangeCertificate) {
-    final Proposal data =
-        messageFactory.createProposal(roundIdentifier, block, roundChangeCertificate);
+    try {
+      final Proposal data =
+          messageFactory.createProposal(roundIdentifier, block, roundChangeCertificate);
 
-    final ProposalMessageData message = ProposalMessageData.create(data);
+      final ProposalMessageData message = ProposalMessageData.create(data);
```

Comment:
The try/catch only needs to be around the messageFactory.create method

Reply:
It could be thinner, but didn't seem to add a lot of value to do so.